style/book.css

```diff
@@ -143,13 +149,13 @@ div.indented {
 
 /* Code Line Types */
 
-.md code > .highlight {
```

Comment:
I don't think this is a CSS syntax error, and prefer spaces around the child selector `>` operator.

Reply:
I agree, I think this looks better with the spaces, so please revert this part back unless this is strictly necessary.